sites/cheerpx/src/content/docs/12-reference/00-CheerpX-Linux-create.md

```diff
@@ -78,4 +78,116 @@ networkInterface?: NetworkInterface;
 
 This option configures network settings, which allows CheerpX to communicate over networks. For more detailed information about how CheerpX handles networking, including the use of Tailscale and overcoming browser limitations, see the [Networking](/docs/guides/Networking) guide.
 
+### `authKey`
+
+```ts
+authKey?: string;
+```
+
+The `authKey` is a string containing an authentication key for registering pre-authenticated users or devices. You can generate one [here](https://login.tailscale.com/admin/settings/keys).
+
+Example:
+
+```js
+const cx = await CheerpX.Linux.create({
+	mounts: mountPoints,
+	networkInterface: { authKey: "YOUR KEY" },
+});
+```
+
+In this example, the `authKey` is included in the `networkInterface` object to enable authentication for the CheerpX instance.
+
+### `controlUrl`
+
+```ts
+controlUrl?: string;
+```
+
+The `controlUrl` is an optional string used to specify the URL of a [self-hosted Headscale server](/docs/guides/Networking#self-hosting-headscale).
+
+Example:
+
+```js
+const cx = await CheerpX.Linux.create({
+	mounts: mountPoints,
+	networkInterface: { controlUrl: "YOUR URL" },
+});
+```
+
+Here, the `controlUrl` is used to point the CheerpX instance to a custom self-hosted server.
```

Comment:
Consider removing these extra comment lines, they are mostly redundant. The example is enough.

Reply:
They can be left if there is actually something to add after the example code